functions/src/mappers/github/pullRequest.mapper.ts

```diff
@@ -28,8 +25,8 @@ export interface GitHubPullRequestModel {
   owner: GitHubUserModel;
   assignees: GitHubUserModel[];
   reviewers: GitHubUserModel[];
-  createdOn: firestore.Timestamp;
-  updatedOn: firestore.Timestamp;
+  createdOn: string;
+  updatedOn: string;
```

Comment:
Why did these change to `string`? I think Firestore has deprecated this in favour of `firestore.Timestamp`

Reply:
For now ‘createdOn’ and ‘updatedOn’ have a string type in our database. If we want to change it to Timestamp we need to create conversion function